### DISCUSSION TITLE
[Factions] The Ending Needs Improvement

### DISCUSSION THREAD

**Discussion by Wheatley1001** [FakeFishGames/Barotrauma#11082] [Unstable feedback]
This discussion will be split into three key sections: environment, lore, and fighting. 

### The Environment 
Let's start with the positive, the environment. Easily one of the most interesting and well crafted sections of the game, the final area brings confusion and intrigue to the crew who comes across it. The background art and structure design is incredibly well done and exploring them both from the Captain's perspective and Dive Team's perspective is a good experience for the player. Although some things could be worked on (the language translator being "???" for the entire game for example. Once its purpose is discovered it could show an actual name on the item), what we have now is something that myself and many others really like. 

### The Lore
Now onto the not so positive. The lore feels forced. Everything important feels slapped in your face immediately in the final mission. Trickled pieces of information spoken by the AI aren't a sufficient replacement for the lack of lore everywhere else. Whilst the Alien Ruins getting a touch-up and nods to the final location did not go unnoticed, it simply isn't enough. Those kinds of subtle hints and nods should be in more locations and should be more visible. Why are they limited to Alien Ruins when we know that there are research stations around Europa that actively investigate this topic? The ending text shows that people clearly know about these ruins and mechanics, yet in the game this is absolutely absent until the final beacon where you meet one scientist who just _happens_ to know everything. What was the point in us bringing all of these alien artifacts to stations for research if it means we learn nothing from them? They've been researching them since before we started as a crew on Europa, so why do we not learn anything from this research? It makes no sense and makes the world feel gutted in terms of genuine development and interaction. 

**Jove**

Jove as a character is nearly completely absent from the world. It isn't in the game physically until the final level where all of a sudden it's there, controls everything, and now you have to kill it. Why? Why do we have to kill it? Why was a character so critical to the story left until the last minute to be introduced? 
Whilst it could be argued that it isn't on Europa and nobody has seen it, that doesn't justify why are there no theories or suspicions of it existing. There are engravings of Jove in the Alien Ruins which are frequently researched and explored by dive teams and research teams, which makes it unbelievable that nobody has figured out what the god-like figure on the walls could be.

It would be dishonest of me to ignore the inclusion of Jove's sounds in one of the music tracks of the game, but the average player would not be able to identify the samples in the song as matching those produced by Jove.

### The Fights
I will be writing about the three main encounters in the game's ending. I will critique the design and combat of each character.

**Cyborg Worm**
**Design**
The Cyborg Worm's design is amazing. It brings a new spin to something every player is aware of and terrified of. Players are able to identify it as an Endworm, yet it's unique enough to bring a new flavour of fear and concern to players experiencing it for the first time. I have nothing negative to write about the design and applaud the choices made for this creature!

**Combat**
The Cyborg Worm fight is easily the best combat situation the player will engage with during the ending location. It has the classic Endworm attacks whilst bringing flavour to the fight with ranged attacks. It forces players to adapt to a fight they're familiar with and tackle it in a way they would have no way of preparing for before their encounter. The fight felt challenging, well fought, and engaging throughout the entire duration of the fight. 

**Fractal Guardians**
**Design**
The use of the fractal guardians as a standard defensive creature throughout the final level ties the alien ruins to the location nicely. It connects the guardians to Jove, which compliments the lore nicely. However, they could benefit from more unique elements to them aside from the weapons and orbs that orbit them. Take the Cyborg Worm as inspiration, for example. Overall the design is good, but could use some minor tweaks to make them stand out even more.

**Combat**
In combat, the Fractal Guardians have many positive traits. They're small, quick, and deal ranged damage against the submarine. However, the rapid deployment of the healing orbs makes them more irritating to fight than engaging. The Guardians will often times deploy multiple of these orbs at once, which turns engagements with the enemy into a stun-locked fight in which the hostile will out-heal the damage against it whilst the gunner refuses to let it heal. The simple solution would be to shoot at the orbs, but as soon as one orb is destroyed another takes its place. I am unsure if this is intentional behaviour or not, but my recommendation would be to tweak the spawn rate of the repair orbs slightly to prevent these situations.

**Jove**
**Design**
Jove is easily one of the best designed creatures in the entire game. Whoever designed the creature, whichever art team or character concept designers came up with Jove's design, they all did outrageously good jobs. There is not a single thing I could possibly say that isn't positive about Jove's design. 

**Combat**
Jove is unfortunately the least engaging creature to fight in the ending area. It has no resistances and 10,000 vitality (the same vitality as the regular Endworm), which makes killing it incredibly quick. The lack of ranged attacks also makes Jove less threatening to a submarine which keeps its distance. Of course, the swarm of Fractal Guardians is something that cannot be ignored, but looking exclusively at Jove's attacks and abilities, the fight is incredibly underwhelming. The attack which triggers if the submarine gets too close is phenomenal, but I wish there were more attacks to that same scale. 
Additionally, the fact that the fight outright ends once Jove is dead kills most of the anticipation and engagement of the fight as gunners can simply target Jove and ignore the Fractal Guardian swarms. There is a lot to be desired out of a final boss fight, and in its current form Jove does not feel like an end boss.

### Closing Statement 
Overall, the ending feels 80% complete. There are overwhelmingly strong aspects of it, but the underwhelming parts take away from this. 1.0's release is rapidly approaching and I fully understand that sweeping changes are unrealistic, but I cannot help but feel concerned that the ending's current state will sour the otherwise stellar update that is the 1.0 release. I have had countless hours of fun, engagement, and joy play testing this build over the last few months and I couldn't sing my praises to the dev team loud enough for the work they've done fixing bugs and adding content to the factions. I've made amazing memories with so many members of the community through this build alone. I desperately want everyone else to experience the same thing I did, but I'm scared the ending will distract from the positive aspects of 1.0 if left unaddressed.

**Comment by Regalis11** [comment]
Thank you for the feedback!

I do agree with many of your opinions, but when it comes to the lore and Jove, I have to point out that it seems you either haven't encountered all the events related to it, or might have somewhat misunderstood some of it (which is no surprise - we intentionally communicate many details about it in vague hints).

> Why are they limited to Alien Ruins when we know that there are research stations around Europa that actively investigate this topic?
> why are there no theories or suspicions of it existing

There are several hints about the scientific community being aware of the ancient aliens worshipping "Jupiter" or some deity associated with Jupiter. You can also meet some characters during the course of the campaign that have actually seen it.

>Why do we have to kill it?

There are also plenty of hints about this, with the conversations in the final level being the most explicit ones.

But in any case, seeing that you've finished the whole campaign and are still not entirely clear on some of the details, it might be a sign that we made many of the hints a bit too subtle or hard to piece together. So thanks for sharing your thoughts!

**Reply by Fentanyl1984** [reply]
its REALLY hard to find any random characters talking about lore details. I can go as far as saying that im the only one who naturally encountered that random colony dweller who actually knows about The Eye and Them(i bet you know what im talking about), although im not 100% sure. And we dont even know how many random characters/events we missed. though im not sure if thats a bad thing. collecting the details of lore piece by piece is a pretty fun activity if you ask me

**Reply by Regalis11** [reply]
That's a good point: we can't expect that every player encounters and talks with all (or even any of) the NPCs that provide details about the ending. So it might make sense for us to make some of the hints harder or impossible to miss.

**Reply by Wheatley1001** [reply]
> There are also plenty of hints about this, with the conversations in the final level being the most explicit ones.

Yeah this is unfortunately a leftover piece of an earlier draft from when I completely misunderstood the ending conversations. Embarrassing that I left it in the final draft tbh... that part is actually quite clear in hindsight.



> There are several hints about the scientific community being aware of the ancient aliens worshipping "Jupiter"

Unfortunately I don't think we encountered any of these in either of the two testing campaigns I hosted. Are they event based, conversation based, or environment based? If they're hidden in the environment then it's absolutely on me for not looking hard enough in research outposts or similar environments, but events might be a bit too rare for players to be able to discover. Conversational lore drops by the AI (crew or outpost) may be too random chance as well.
I don't want the lore or anyone's story to be spoonfed to players or observers, it just doesn't feel present a lot of the time. If the story is to be told through subtle hints and riddles to symbolise the overall confusion behind the entity and what's going on, then it would benefit from having these show more frequently so players don't accidentally dismiss these as random AI messages or events that don't progress the story or campaign (for example, the outpost fight event. It's a great event, but has no impact to the story or campaign progression).

**Comment by soledad630** [comment]
A small point to add, in my own playthrough I managed to be able to wait out the full timer of volatile compound N that I spawned in against jove because I parked outside of the range of his balls and he is trapped in rocks. But based on his speed who's to say we cannot do that in normal situations, fire at his general direction, and observe health bar change to basically cheese him with range superiority.

His energy balls has to have infinite reach or he's very easily cheeseable since we also gets his exact location based on the sub on the reveal cutscene and pylon destruction cutscene, making the initial calibration incredibly easy.

**Reply by NotWendy** [reply]
I think the best-case scenario would be something coming from the level's borders forcing the players to get closer to Jove and stay within his effective range. Even something as simple as a strong water current which cuts off some distance before Jove would make the fight much harder to cheese. And it'd be clearly visible on the sonar too.

**Comment by Ek-Videogames** [comment]
I've kind of felt that same love-hate relationship with the lore ever since "the europan" premium DLC.

We get all kinds of colorful lore and worldbuilding, but very little of it has any meaningful in-game implications besides gimmick quest-rewards, and what little is there feels - as the post author says - like forced and overly self-referential. Its like the game is saying "Hey remember this thing" but don't think too hard about it...
